float64[] act_x_pos
float64[] act_y_pos
float64[] true_x_pos
float64[] true_y_pos
float64[] traj_x_pos
float64[] traj_y_pos